backend/http_settings.go

```diff
@@ -273,7 +274,7 @@ func parseHTTPSettings(jsonData json.RawMessage, secureJSONData map[string]strin
 
 		if key, exists := dat[headerNameSuffix]; exists {
 			if value, exists := secureJSONData[headerValueSuffix]; exists {
-				s.Headers[key.(string)] = value
+				s.Header.Add(key.(string), value)
```

Comment:
Don't understand why key needs to be cast to a string - isn't it already a string since `dat` is `map[string]interface{}`?

Reply:
`dat` is `map[string]interface{}`, so `key := dat[headerNameSuffix]` is `interface{}`.

On the other side `secureJSONData` is `map[string]string`, so `value doesn't need cast.

Reply:
Oh yeah I see that now :+1: